scala/src/main/scala/smile/clustering/k-modes/ScalaClusteringTypes.scala

```diff
@@ -0,0 +1,10 @@
+package smile.clustering
+
+trait ScalaClusteringTypes
```

Comment:
Why this trait?

Reply:
At origin it is because i have multiple clustering algorithms sharing this common properties. It could be more usefull when i will add the others algorithms.
I removed it for the moment.